translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/database/ProjectDatabaseHelper.java

```diff
@@ -1140,6 +1140,26 @@ public int getProjectProgressSum(int projectId) {
         return progress;
     }
 
+    public int getProjectProgress(int projectId) {
+        String projectIdString = String.valueOf(projectId);
+        SQLiteDatabase db = getReadableDatabase();
+        final String query = String.format("SELECT %s FROM %s WHERE %s=?",
+                ProjectContract.ProjectEntry.PROJECT_PROGRESS, ProjectContract.ProjectEntry.TABLE_PROJECT, ProjectContract.ProjectEntry._ID);
```

Comment:
when you have to split arguments onto another line because of length (and this still has line length issues), each parameter goes on its own line like so:

```
 final String query = String.format(
    "SELECT %s FROM %s WHERE %s=?",
    ProjectContract.ProjectEntry.PROJECT_PROGRESS, 
    ProjectContract.ProjectEntry.TABLE_PROJECT, 
    ProjectContract.ProjectEntry._ID
);
```